frontend/packages/gitops-plugin/src/components/details/GitOpsResourceRow.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { Split, SplitItem, Tooltip } from '@patternfly/react-core';
+import { HeartBrokenIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import {
+  global_danger_color_100 as RedColor,
+  global_warning_color_100 as YellowColor,
+} from '@patternfly/react-tokens';
+import { useTranslation } from 'react-i18next';
+import { GitOpsEnvironmentService, GitOpsHealthResources } from '../utils/gitops-types';
+import './GitOpsResourcesSection.scss';
+
+interface GitOpsResourceRowProps {
+  resources: GitOpsHealthResources[] | GitOpsEnvironmentService[];
+  degradedResources: string[] | null;
+  nonSyncedResources: string[];
+}
+
+const GitOpsResourceRow: React.FC<GitOpsResourceRowProps> = ({
+  resources,
+  degradedResources,
+  nonSyncedResources,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Split hasGutter>
+      {degradedResources?.length > 0 && (
+        <Tooltip
+          content={t('gitops-plugin~{{x}} of {{total}} Unhealthy', {
+            x: degradedResources.length.toString(),
+            total: resources?.length.toString() ?? '0',
+          })}
+        >
+          <SplitItem>
+            <>
+              {degradedResources.length}
+              &nbsp;
+              <HeartBrokenIcon color={RedColor.value} className="co-icon-space-r" />
+            </>
+          </SplitItem>
+        </Tooltip>
+      )}
+      {nonSyncedResources.length > 0 && (
+        <Tooltip
+          content={t('gitops-plugin~{{x}} of {{total}} OutOfSync', {
+            x: nonSyncedResources.length.toString(),
+            total: resources?.length.toString() ?? '0',
+          })}
+        >
+          <SplitItem>
+            <>
+              {nonSyncedResources.length}
+              &nbsp;
+              <ExclamationTriangleIcon color={YellowColor.value} className="co-icon-space-r" />
+            </>
+          </SplitItem>
+        </Tooltip>
+      )}
+      {degradedResources?.length === 0 && nonSyncedResources.length === 0 && <>&nbsp;</>}
```

Comment:
nit: 
```suggestion
      {degradedResources?.length === 0 && nonSyncedResources.length === 0 && ' '}
```

Reply:
Thanks for the review and feedback. Here is different from the other two spaces, here it needs to take an empty row instead of space between two elements in same row. Please refer the below screenshot. If I make this change, it will stack to top of the column where I put 'Degraded' and 'OutOfSync' for 'Routes'.
<img width="340" alt="Screen Shot 2021-12-07 at 2 23 03 PM" src="https://user-images.githubusercontent.com/26255262/145094131-98e861fe-563e-4751-90b7-b1cbaad71a17.png">